### ISSUE TITLE
Add ability to filter channels using service id

### ISSUE THREAD

**Issue by vsbogd** [singnet/snet-cli#151]
At the moment `snet-cli print_all_channels` and `snet-cli print_initialized_channels` command return the full list of channels but if client used few different services and he is looking for the some specific services it should be a way to filter out channels which are related to specific service.

**Comment by astroseger** [comment]
I've added it in #148.
We will be able to filter channels by given service metadata file with ```--metadata_file``` options, or by given service registration with ```--org_id``` and ```--service_id``` (they should be specified together).
For example
```
# print all initilized channels which has the current identity as a sender and corresponds to service_metadata.json
snet client print_initialized_channels --metadata_file service_metadata.json

# print all channels which has the current identity as a sender and have been opened for testo/tests2
snet client print_all_channels --org_id testo --service_id tests2
```
However I've realized that  for ```print_all_channels``` it hasn't been implemented in the most efficient way, because I could filter ethereum events directly with group_id. 

So we can leave this issue opened after #148 for possible optimization...

**Comment by astroseger** [comment]
I've changed my solution in #148. Not, I believe,  it solves this issue better.
I've added the following functions:
1. ```snet channel print-initialized```  for print initialized channels. By default it prints all channels. It have the following optional parameters:
      * ```--only-id```             Print only id of channels
      * ```--filter-sender```       Print only channels in which current identity is sender
      *  ```--filter-signer```       Print only channels in which current identity is signer
      * ```--filter-my```           Print only channels in which current identity is  sender or signer

2. ```snet channel  print-initialized-filter-group``` Print initialized channels for the given service (given payment group). User should specify org_id and service_id (and --group-name in case of multiply group service). It have the same optional parameters. 

3.  ```snet channel print-all-filter-sender```  Print all channels for the given sender (from blockchain). 

4. ```snet channel print-all-filter-recipient```  Print all channels for the given recipient (from blockchain!)

5. ```snet channel print-all-filter-group```  Print all channels for the given service. User should specify org_id and service_id (and --group-name in case of multiply group service). This function might be useful for service providers, but not for the "clients".

6.  ```snet channel print-all-filter-group-sender```  Print all channels for the given group **and** sender. User should specify org_id and service_id (and --group-name in case of multiply group service). This function is useful for the clients.

**Comment by astroseger** [comment]
was closed in #148
